src/Community.VisualStudio.Toolkit.Shared/Solution/Solutions.cs

```diff
@@ -161,78 +162,84 @@ private async Task<IEnumerable<IVsHierarchyItem>> GetSelectedHierarchiesAsync()
             try
             {
                 svc.GetCurrentSelection(out hierPtr, out uint itemId, out IVsMultiItemSelect multiSelect, out containerPtr);
+                await AddHierarchiesFromSelectionAsync(hierPtr, itemId, multiSelect, results);
```

Comment:
ℹ️ The diff looks a bit weird here, but I've refactored the code that turns the result of `GetCurrentSelection` into `IVsHierarchy` objects into a separate method because `SolutionExplorerWindow.GetSelectionAsync` needed to do exactly the same thing. Hiding whitespace changes makes it a bit easier to see what changed.